assignments/arrays.js

```diff
@@ -63,34 +63,58 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
 
+for (let i = 0; i< inventory.length; i++) {
+    if(inventory[i].id === 33){
+        console.log(`Car 33 is a ${inventory[i].car_year} ${inventory[i].car_make} ${inventory[i].car_model}` );
```

Comment:
This is great.  Look into the `break;` statement.  If you had that in your if check you'd save your computer a few calculations.